standalone/lcio2edm4hep.cpp

```diff
@@ -204,10 +204,11 @@ int main(int argc, char* argv[]) {
 
   const int nEvt = args.nEvents > 0 ? args.nEvents : lcreader->getNumberOfEvents();
   bool haveSimCaloHits{false};
+  const int tenPercent = nEvt / 10;
   for (int i = 0; i < nEvt; ++i) {
-    int percEvt = i * 100 / (nEvt - 1);
-    if (percEvt % 10 == 0) {
-      std::cout << "processed amount of events: " << percEvt << "% (event: " << i << ")" << std::endl;
+    if ((i + 1) % tenPercent == 0) {
```

Comment:
If `nEvt < 10` then `tenPercent` is zero and `% 0` can't be done.

Reply:
```suggestion
    if (tenPercent && ((i + 1) % tenPercent == 0)) {
```